doc/kubernetes/modules/ROOT/pages/running/metrics/ispn_metrics.adoc

```diff
@@ -0,0 +1,242 @@
+= {jdgserver_name} Metrics
+:description: This document contains details of the {jdgserver_name} metrics that can be used to monitor your deployment's performance
+
+include::partial$running/infinispan-attributes.adoc[]
+
+{description}
+
+
+====
+*Global tags*
+
+* `cache=<name>`: The cache name.
```

Comment:
Would it make sense to split this file to per-cache metrics and global/per-node metrics? If I am not mistaken, not all metrics contain the cache tag as mentioned here.

Reply:
I'm not adding any global metrics to this file because I don't see the relevance.
Do you have any global metric in mind that can be useful?

Reply:
Sorry, I thought that some metrics in this file do not have cache tag, like the one about state transfer or transactions but I checked the today's clusters and it seems they all have. Thanks for the answer.